save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -108,7 +108,7 @@ class DownloadFilesController(
     fun uploadImage(
         @RequestPart("file") file: Mono<FilePart>,
         @RequestParam owner: String,
-        @RequestParam(required = false) type: AvatarType = AvatarType.ORGANIZATION
+        @RequestParam type: AvatarType
```

Comment:
why without a default value?

Reply:
It doesn't make sense. It will be strange to upload an avatar to an organization if we have not indicated what it is for.